src/components/CellPanel/CellPanel.tsx

```diff
@@ -56,6 +57,6 @@ export default function CellPanel({ df, row, col, setProgress, setError, onClose
   </>
 
   return <ContentWrapper headers={headers}>
-    <code className="text">{text}</code>
+    <code className={styles.textView}>{text}</code>
```

Comment:
wow, well spotted. Thanks for fixing it

Reply:
Hmmm, we still have an issue here:

https://github.com/hyparam/hyperparam-cli/blob/9b622ef7bcc7c86a73dcaeeb82529d438d2aaaee/src/components/Cell/Cell.tsx#L72

I'll send a PR

Reply:
https://github.com/hyparam/hyperparam-cli/pull/216